NitroxClient/GameLogic/Spawning/WorldEntities/PlaceholderGroupWorldEntitySpawner.cs

```diff
@@ -21,12 +24,17 @@ public PlaceholderGroupWorldEntitySpawner(WorldEntitySpawnerResolver spawnerReso
         this.defaultSpawner = defaultSpawner;
     }
 
+    // This spawner can't hold a SpawnSync function because it is also responsible for spawning its children
```

Comment:
Would elevate this to a proper xml comment as this method does something against it's name (bc the name can't be changed, is not designed to change)